src/components/bottom-sheet/BottomSheetHeader.tsx

```diff
@@ -0,0 +1,106 @@
+import {LayoutChangeEvent, TouchableOpacity, View} from 'react-native';
+import React, {useState} from 'react';
+import {StyleSheet, useTheme} from '@atb/theme';
+import {ThemeText} from '@atb/components/text';
+import {ThemeIcon} from '@atb/components/theme-icon';
+import {useBottomSheet} from '@atb/components/bottom-sheet';
+import {BottomSheetTexts, useTranslation} from '@atb/translations';
+import {Close} from '@atb/assets/svg/mono-icons/actions';
+
+type BottomSheetHeaderWithoutNavigationProps = {
+  title?: string;
+  closeBottomSheet?: () => void;
+};
+
+// Define the interface for the size
+interface Size {
+  width: number;
+  height: number;
+}
+
+export const BottomSheetHeader = ({
+  title,
+  closeBottomSheet,
+}: BottomSheetHeaderWithoutNavigationProps) => {
+  const styles = useStyle();
+  const {t} = useTranslation();
+
+  const {theme} = useTheme();
+  const {close: closeBottomSheetDefault, onOpenFocusRef} = useBottomSheet();
+
+  const handleClose = closeBottomSheet ?? closeBottomSheetDefault;
+
+  const themeColor = 'interactive_3';
+  const {background: backgroundColor, text: textColor} =
+    theme.interactive[themeColor]['active'];
+
+  const [buttonSize, setButtonSize] = useState<Size>({width: 0, height: 0});
+  const onButtonLayout = (event: LayoutChangeEvent) => {
+    const {width, height} = event.nativeEvent.layout;
+    setButtonSize({width, height});
+  };
+
+  // Calculate hitSlop values
+  const desiredHitboxSize = 48;
+  const hitSlopSize = (desiredHitboxSize - buttonSize.height) / 2;
+
+  const hitSlop = {
+    top: hitSlopSize,
+    bottom: hitSlopSize,
+    left: hitSlopSize,
+    right: hitSlopSize,
+  };
```

Comment:
Shouldn't overthink this hitslop stuff, it is not worth the effort. Just remove it all and use `hitSlop={insets.all(10)}` on the `TouchableOpacity`.